bundles/bundle.properties

```diff
@@ -40,151 +40,158 @@ block.btm-T2-swarmer.name = T2 Swarmer
 block.btm-heal.name = Repair Projector Turret
-block.btm-heal.description = Projection repair buildings and units in the area
+block.btm-heal.description = Projection repair buildings and units in the area.
 block.btm-turret-resupply-point.name = Turret Resupply Point
 block.btm-turret-resupply-point.description = Refill the turret with designated ammunition within a certain range
 
 block.btm-ar-node.name = Plastanium Armor Node
-block.btm-ar-node.description = stronger than power node, but at the expense of the number of connections\nCan be placed on liquid
+block.btm-ar-node.description = stronger than power node, but at the expense of the number of connections\nCan be placed on deep liquids.
 block.btm-ar-node.details = Used the buoyancy of plastanium to float on liquid(No, it's not like that...
 block.btm-power-node-giant.name = Power Center
-block.btm-power-node-giant.description = Gathering the power of batteries, power nodes.
-block.btm-T2-steam-generator.name = advanced-steam-generator
+block.btm-power-node-giant.description = An advanced power node with even greater range and power storage ability.
+block.btm-T2-steam-generator.name = Advanced Steam Generator
 block.btm-T2-steam-generator.description = More and faster consumption, but can produce a lot of electricity.
 block.btm-T2ther.name = Thermal Reactor
 block.btm-T2ther.description = Powerful thermal generator, But it takes time to activate.
-block.btm-Th2.name = thorium reactor ii
-block.btm-Th2.description = Consume more but can output a lot of electricity.Have a certain defense
+block.btm-Th2.name = Thorium Reactor MKII
+block.btm-Th2.description = Consume more but can output a lot of electricity.Have a certain defense.
 block.btm-lightnin-generator.name = Lightnin Reactor
-block.btm-lightnin-generator.description = Consumes the Lightnin Alloy and releases a large amount of electricity
+block.btm-lightnin-generator.description = Use Lightnin Alloy and produce a large amount of energy.
+
 block.btm-conduit.name = Phase Conduit
-block.btm-conduit.description = Faster and can be placed on the liquid
-block.btm-TLR.name = titanium Liquid Router
+block.btm-conduit.description = Faster and can be placed on deep liquids.
+block.btm-TLR.name = Titanium Liquid Router
 block.btm-TLR.description = Faster, stronger!
 block.btm-TLB.name = Titanium Bridge Conduit
 block.btm-TLB.description = farther，stronger!
 block.btm-lb.name = Liquid Node
-block.btm-lb.description = Pass the liquid to the further end.[yellow]AUTOMATIC CONNECTION
+block.btm-lb.description = Pass the liquid to the other end.[yellow]AUTOMATIC CONNECTION
 block.btm-ld.name = Liquid Mass Driver
 block.btm-ld.description = For long-distance transmission of liquids.
+
 block.btm-alloy-duct.name = Surge Duct
-block.btm-alloy-duct.description = Faster speed and can be placed on the liquid
+block.btm-alloy-duct.description = Faster speed and can be placed on deep liquids.
```

Comment:
weird expression
nearly **ANY** building can be place on sand water or ordinary water tile
it should be **deep** liquids